hermetic_build/DEVELOPMENT.md

```diff
@@ -160,13 +160,15 @@ python hermetic_build/library_generation/cli/entry_point.py generate \
 1. Run the following command to build the image from source
 
    ```shell
-   docker build \
+   DOCKER_BUILDKIT=1 docker build \
      -f .cloudbuild/library_generation/library_generation.Dockerfile \
      -t local:image-tag \
      .
    ```
+   Please note that the build only works when using the new
+   [Docker BuildKit](https://docs.docker.com/build/buildkit/) (enabled through the `DOCKER_BUILDKIT` variable).
    
-2. Set the version of gapic-generator-java
+3. Set the version of gapic-generator-java
```

Comment:
Is this the right num?

Reply:
```suggestion
2. Set the version of gapic-generator-java
```

I didn't see the editor changed the number. Thanks for the catch